shared/postgres/scripts/insert_test_employees.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Copyright © 2025 Meroxa, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+. "$SCRIPT_DIR/common.sh"
+
+cleanup() {
+  echoerr "Interrupted. Exiting..."
+  exit
+}
+
+trap cleanup SIGINT
+
+# Number of batches
+BATCHES=$1
+
+# Number of records per batch
+BATCH_SIZE=$2
```

Comment:
It might be simpler to reason about the script if it accepted just the total number of rows to be inserted.

Reply:
@hariso why not having the flexibility of being able to configure the batches?

Reply:
I can see that 👍